src/main/java/org/sonar/plugins/stash/coverage/CoverageProjectStore.java

```diff
@@ -0,0 +1,57 @@
+package org.sonar.plugins.stash.coverage;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.sonar.api.BatchComponent;
+import org.sonar.api.batch.InstantiationStrategy;
+import org.sonar.api.batch.Sensor;
+import org.sonar.api.batch.SensorContext;
+import org.sonar.api.batch.rule.ActiveRules;
+import org.sonar.api.resources.Project;
+import org.sonar.plugins.stash.StashPluginConfiguration;
+import org.sonar.wsclient.Sonar;
+
+import static org.sonar.plugins.stash.coverage.CoverageUtils.calculateCoverage;
+import static org.sonar.plugins.stash.coverage.CoverageUtils.createSonarClient;
+import static org.sonar.plugins.stash.coverage.CoverageUtils.getLineCoverage;
+
+@InstantiationStrategy(InstantiationStrategy.PER_BATCH)
+public class CoverageProjectStore implements BatchComponent, Sensor {
+    private static final Logger LOGGER = LoggerFactory.getLogger(CoverageProjectStore.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "LOGGER" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)